engine/tests/phpunit/integration/Elgg/Integration/ElggCoreAccessCollectionsTest.php

```diff
@@ -25,49 +23,6 @@ public function up() {
 		elgg()->session->setLoggedInUser($this->user);
 	}
 
-	public function testUpdateACL() {
```

Comment:
don't we need these tests?

Reply:
check the commit a8a9bf43b7d613d4bcd61a163ede66862d5efa4d